cmd/p4info_code_gen/p4info_code_gen.go

```diff
@@ -220,11 +223,30 @@ func generateConstants(p4info *p4ConfigV1.P4Info) string {
 		constBuilder.WriteString(emitEntitySizeConstant(mtrSizeVarPrefix, name, element.GetSize()))
 	}
 
+	// Enums
+	constBuilder.WriteString("// Enumerators\n")
+	for eName, value := range p4info.GetTypeInfo().GetSerializableEnums() {
+		for _, member := range value.GetMembers() {
+			name := eName + "_" + member.GetName()
+			if len(member.GetValue()) > 4 {
+				log.Errorln(name, " cannot fit in Uint32")
+				continue
+			}
+			constBuilder.WriteString(emitEntityConstant(enumVarPrefix, name, getUint32FromByteArray(member.GetValue())))
+		}
+	}
+
 	constBuilder.WriteString(constOrVarClose + "\n")
 
 	return constBuilder.String()
 }
 
+func getUint32FromByteArray(s []byte) uint32 {
+	var b [4]byte
+	copy(b[4-len(s):], s)
+	return binary.BigEndian.Uint32(b[:])
+}
```

Comment:
```suggestion
func getUint32FromByteArray(s []byte) uint32 {
	return binary.BigEndian.Uint32(s[4-len(s):])
}
```

Reply:
Tried as suggested, but fails because `binary.BigEndian.Uint32` expects a 4 bytes array, so we would need to add padding when `s` is shorter than 4 bytes.